pyne/ace.pyx

```diff
@@ -110,6 +111,79 @@ def ascii_to_binary(ascii_file, binary_file):
     # Close binary file
     binary.close()
 
+def _interpolation_tab1(xin, x, y, interp_NBT = None, interp_INT = None):
+    """
+     INTERPOLATE_TAB1 interpolates a function between two points based on
+     particular interpolation scheme. The data needs to be organized as a ENDF TAB1
+     type function containing the interpolation regions, break points, and
+     tabulated x's and y's.
+    """
+    
+    cdef int i, interp  
+    if xin <= x[0]:
+        return y[0]
+    elif xin >= x[-1]:
+        return y[-1]
+    else:
+        I = np.searchsorted(x, xin) - 1
+    
+    # Determine interpolation scheme 
+    if interp_NBT == None:
```

Comment:
You should almost never do `== None` in Python.  It is horrendously slow and not really want you want.  Please change this to `is None`. Thanks!